virl/files/mitaka.bashrc

```diff
@@ -120,7 +120,7 @@ export OS_USERNAME=admin
 export OS_PASSWORD=ADMIN_PASS
 export OS_TENANT_NAME=admin
 export OS_AUTH_URL=http://controller:35357/v3
-export OS_SERVICE_ENDPOINT=http://controller:35357/v3
+export OS_SERVICE_ENDPOINT=http://controller:35357/v2.0
```

Comment:
The keystone CLI client is apparently broken with v3, with the solution recommended 'just move to openstack'. We can likely do that with all uses in salt, and probably from (troubleshooting) documentation, yet I'm not sure whether no one else uses that cli.